src/mnode/rpc/masternode.cpp

```diff
@@ -55,11 +55,20 @@ UniValue masternodelist(const UniValue& params, bool fHelp)
 {
     std::string strMode = "status";
     std::string strFilter;
+    std::string strExtra = "";
```

Comment:
You don't need to assign "" to strExtra, string default constructor sets it to empty.